src/org/opendatakit/briefcase/util/ExportToCsv.java

```diff
@@ -130,11 +129,9 @@ public boolean doAction() {
       return false;
     }
 
-    File[] instances = instancesDir.listFiles(new FileFilter() {
-      public boolean accept(File file) {
-        // do we have a folder with submission.xml inside
-        return file.isDirectory() && new File(file, "submission.xml").exists();
-      }
+    File[] instances = instancesDir.listFiles(file -> {
```

Comment:
This could be:
```java
// do we have a folder with submission.xml inside
File[] instances = instancesDir.listFiles(file -> file.isDirectory() && new File(file, "submission.xml").exists());
```